src/main/sensors/esc_sensor.c

```diff
@@ -349,4 +349,9 @@ void escSensorProcess(timeUs_t currentTimeUs)
             break;
     }
 }
+
+uint16_t calcRpmFromErpm(int16_t eRPM)
```

Comment:
The name of this function doesn't illustrate the fact that it's not actually `eRPM / 100` which is being converted.
Looking at other places, we don't capitalize acronyms, so it should probably be `int16_t erpm`